src/Components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { DropdownPanel } from "../components";
+import "./VideoCard.css";
+
+function VideoCard({ video }) {
+  console.log("And here?");
+  console.log(video);
```

Comment:
you can remove console

Reply:
Thant You. Will add Toast with other features